pkg/agent/proxy/metrics/metrics.go

```diff
@@ -0,0 +1,117 @@
+package metrics
+
+import (
+	"sync"
+
+	kmetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog"
+)
+
+var (
+	once sync.Once
+
+	subsystemProxy = "antrea_proxy"
+
+	SyncProxyLatency = kmetrics.NewHistogram(
+		&kmetrics.HistogramOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "sync_proxy_rules_duration_seconds",
+			Help:        "SyncProxyRules latency in seconds of Antrea Proxy",
+			Buckets:     kmetrics.ExponentialBuckets(0.001, 2, 15),
+		},
+	)
+	ServicesInstalledTotal = kmetrics.NewGauge(
+		&kmetrics.GaugeOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "total_services_installed",
+			Help:        "The cumulative number of Services installed at Antrea Proxy",
```

Comment:
"cumulative" seems to be overly used. In this context, it may mean that we consider the services that are installed even though they are deleted later.

Will you consider changing the description to "The number of Service installed"? Similarly for other guage metric.